### ISSUE TITLE
Improve user interface

### ISSUE THREAD

**Issue by sils1297** [coala-analyzer/coala#2207]
would be good to show the current result only, then clear the screen for the next result to focus.

CC @mr-karan that could be part of your project?

**Comment by mr-karan** [comment]
Awesome, I'll look into how to achieve this. Will add to Gitlab milestones

**Comment by jayvdb** [comment]
@sils @mr-karan , I am not sure I follow this.  A CLI issue.
is there any more context to this?

Adding cli paging support?
ncurses on linux?

One result only?  Then press a key to continue?

**Comment by sils** [comment]
yeah the idea was to have a curses like interface where we just show one result after another so it's always focused on the latest result. Press a key to continue.

**Comment by jayvdb** [comment]
ok, I've changed the title to be ncurses.  obviously we'll need something cross-platform, but at least that issue title helps us know what this is about
